charts/hivemq-platform/tests/services/hivemq_metrics_services_test.yaml

```diff
@@ -1056,3 +1056,26 @@ tests:
       - notFailedTemplate: {}
       - hasDocuments:
           count: 1
+
+  - it: with a Metrics service enabled and HiveMQ listener name enabled, validation fails
+    set:
+      services:
+        - type: metrics
+          exposed: true
+          containerPort: 9399
+          hivemqListenerName: foobar
+    asserts:
+      - failedTemplate:
+          errorPattern: Service type `metrics` with container port `9399` is using `hivemqListenerName` value. HiveMQ listener names are only supported by MQTT and WebSocket services
```

Comment:
Container port?

Reply:
Same comment is applicable here: https://github.com/hivemq/helm-charts/pull/403#discussion_r1866225099